core-plugins/docs/OracleExport-action.md

```diff
@@ -0,0 +1,76 @@
+# Oracle Export Action
+
+
+Description
+-----------
+A Hydrator Action plugin to efficiently export data from Oracle to HDFS or local file system.
+The plugin uses Oracle's command line tools to export data.
+The data exported from this tool can then be used in Hydrator pipelines.
+
+
+Use Case
+--------
+A Hydrator user would like to export oracle data onto HDFS or local file system using an action plugin
+that exports Oracle data using the Oracle bulk export tool as opposed to using JDBC.
+
+
+Properties
+----------
+
+**oracleServerHostname:** Hostname of the remote Oracle Host.
+
+**oracleServerSSHPort:** Port to use to SSH to the remote Oracle Host. Defaults to 22.
+
+**oracleServerUsername:** Username to use to connect to the remote Oracle Host via SSH.
+
+**oracleServerPassword:** Password to use to connect to the remote Oracle Host via SSH.
+
+**dbUsername:** Username to connect to the Oracle database.
+
+**dbPassword:** Password to connect the Oracle database.
+
+**oracleHome:** Absolute path of the ``ORACLE_HOME`` environment variable on the Oracle server host.
+This will be used to run the Oracle Spool utility.
+
+**oracleSID:** Oracle System ID(SID). This is used to uniquely identify a particular database on the system.
+
+**queryToExecute:** Query to be executed to export.For example: ``select * from test where name='cask'``;
+
+**tmpSQLScriptDirectory:** Path to the directory where temporary SQL script needs to be created. It will be removed
+once the SQL command is executed.  Default directory is /tmp.
+
+**outputPath:** Path where output file will be exported.
```

Comment:
should mention that this can be an HDFS path. Also, should mention some examples: what happens when there is no scheme/host/port in the URI, only path, etc.